### ISSUE TITLE
IntelliJ plugin error (Gauge v0.3.2)

### ISSUE THREAD

**Issue by Lyxnx** [getgauge/gauge#610]
### Actual behavior
Load up IntelliJ 2017.1, presented with an error:
```
Could not find executable in `GAUGE_HOME`, `PATH` or `GAUGE_ROOT`
java.lang.Throwable
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:132)
	at com.thoughtworks.gauge.util.GaugeUtil.getPath(GaugeUtil.java:75)
	at com.thoughtworks.gauge.util.GaugeUtil.getGaugeExecPath(GaugeUtil.java:51)
	at com.thoughtworks.gauge.core.GaugeVersion.getVersion(GaugeVersion.java:36)
	at com.thoughtworks.gauge.core.GaugeVersion.<clinit>(GaugeVersion.java:27)
	at com.thoughtworks.gauge.GaugeModuleComponent.moduleAdded(GaugeModuleComponent.java:93)
	at com.intellij.openapi.module.impl.ModuleImpl.moduleAdded(ModuleImpl.java:211)
	at com.intellij.openapi.module.impl.ModuleManagerImpl.a(ModuleManagerImpl.java:575)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:1023)
	at com.intellij.openapi.module.impl.ModuleManagerImpl.fireModuleAddedInWriteAction(ModuleManagerImpl.java:573)
	at com.intellij.openapi.module.impl.ModuleManagerComponent.c(ModuleManagerComponent.java:129)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:86)
	at com.intellij.openapi.application.TransactionGuardImpl.submitTransactionAndWait(TransactionGuardImpl.java:151)
	at com.intellij.openapi.module.impl.ModuleManagerComponent.fireModulesAdded(ModuleManagerComponent.java:129)
	at com.intellij.openapi.module.impl.ModuleManagerImpl$1.projectOpened(ModuleManagerImpl.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.util.messages.impl.MessageBusConnectionImpl.deliverMessage(MessageBusConnectionImpl.java:117)
	at com.intellij.util.messages.impl.MessageBusImpl.doPumpMessages(MessageBusImpl.java:437)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpWaitingBuses(MessageBusImpl.java:398)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpMessages(MessageBusImpl.java:387)
	at com.intellij.util.messages.impl.MessageBusImpl.sendMessage(MessageBusImpl.java:373)
	at com.intellij.util.messages.impl.MessageBusImpl.access$200(MessageBusImpl.java:44)
	at com.intellij.util.messages.impl.MessageBusImpl$2.invoke(MessageBusImpl.java:230)
	at com.sun.proxy.$Proxy23.projectOpened(Unknown Source)
	at com.intellij.openapi.project.impl.ProjectManagerImpl$1.projectOpened(ProjectManagerImpl.java:95)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.k(ProjectManagerImpl.java:686)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.e(ProjectManagerImpl.java:357)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:163)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:86)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:109)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.a(LaterInvocator.java:410)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:399)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.d(IdeEventQueue.java:827)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:655)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:365)
	at com.intellij.ide.IdeEventQueue.pumpEventsForHierarchy(IdeEventQueue.java:923)
	at com.intellij.openapi.progress.util.ProgressWindow.startBlocking(ProgressWindow.java:199)
	at com.intellij.openapi.progress.util.ProgressWindow.startBlocking(ProgressWindow.java:186)
	at com.intellij.openapi.application.impl.ApplicationImpl.runProcessWithProgressSynchronously(ApplicationImpl.java:608)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcessWithProgressSynchronously(CoreProgressManager.java:422)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.runProcessWithProgressSynchronously(ProgressManagerImpl.java:89)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:337)
	at com.intellij.openapi.progress.impl.CoreProgressManager.b(CoreProgressManager.java:339)
	at com.intellij.openapi.progress.impl.CoreProgressManager.run(CoreProgressManager.java:323)
	at com.intellij.openapi.progress.ProgressManager.run(ProgressManager.java:190)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.loadAndOpenProject(ProjectManagerImpl.java:437)
	at com.intellij.ide.impl.ProjectUtil.openProject(ProjectUtil.java:189)
	at com.intellij.ide.RecentProjectsManagerImpl.doOpenProject(RecentProjectsManagerImpl.java:60)
	at com.intellij.ide.RecentProjectsManagerBase.doReopenLastProject(RecentProjectsManagerBase.java:619)
	at com.intellij.ide.RecentProjectsManagerBase$MyAppLifecycleListener.appStarting(RecentProjectsManagerBase.java:661)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.util.messages.impl.MessageBusConnectionImpl.deliverMessage(MessageBusConnectionImpl.java:117)
	at com.intellij.util.messages.impl.MessageBusImpl.doPumpMessages(MessageBusImpl.java:437)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpWaitingBuses(MessageBusImpl.java:398)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpMessages(MessageBusImpl.java:387)
	at com.intellij.util.messages.impl.MessageBusImpl.sendMessage(MessageBusImpl.java:373)
	at com.intellij.util.messages.impl.MessageBusImpl.access$200(MessageBusImpl.java:44)
	at com.intellij.util.messages.impl.MessageBusImpl$2.invoke(MessageBusImpl.java:230)
	at com.sun.proxy.$Proxy58.appStarting(Unknown Source)
	at com.intellij.idea.IdeaApplication$IdeStarter.a(IdeaApplication.java:350)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:86)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:109)
	at com.intellij.openapi.application.TransactionGuardImpl.submitTransaction(TransactionGuardImpl.java:118)
	at com.intellij.openapi.application.TransactionGuard.submitTransaction(TransactionGuard.java:122)
	at com.intellij.idea.IdeaApplication$IdeStarter.main(IdeaApplication.java:348)
	at com.intellij.idea.IdeaApplication.a(IdeaApplication.java:202)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:193)
	at com.intellij.idea.IdeaApplication.run(IdeaApplication.java:202)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.idea.MainImpl$1$1.a(MainImpl.java:63)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:343)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

### Steps to reproduce
1. Occurs on IDE load

### Gauge version
```
IntelliJ 2017.1 plugin v0.3.2
```

**Comment by davef800** [comment]
This same exeption just happened to me as well for the same versions described in comment above after upgrading to IntelliJ 2017.1

**Comment by Lyxnx** [comment]
Glad to know I'm not the only one

**Comment by Apoorva-GA** [comment]
Hi,
Could you please install the latest Gauge (0.8.2) and try again.

**Comment by mikesmth** [comment]
Yup, updating gauge worked for me.
brew install gauge # on Mac-OS ;-) 
https://getgauge.io/get-started.html

**Comment by djmorison** [comment]
According to this page:

https://plugins.jetbrains.com/plugin/7535-gauge

The latest version is 0.3.2, and this is the version specified in the original question, therefore there is no option currently to upgrade.

I am still having this problem.

**Comment by kashishm** [comment]
@djmorison 0.3.2 is the latest version for Gauge IntelliJ plugin. Please update [Gauge(the command line utility)](https://getgauge.io/get-started.html) in your machine.

You can check the Gauge version by running `gauge -v` in the terminal. After updating, it should give something similar to the following output
```
Gauge version: 0.8.2

Plugins
-------
html-report (3.1.0)
```

**Comment by djmorison** [comment]
@kashishm I thought this page was only talking about the IntelliJ plugin.  I don't have the command line tool installed:

```
gauge -v
-bash: gauge: command not found
```

**Comment by kashishm** [comment]
@djmorison Thanks for the info. We have released Gauge IntelliJ plugin 0.3.3 with the fix. Let us know if the issue still persists.

**Comment by kashishm** [comment]
@djmorison Please reopen the issue if you still get the exception.
